students/1/ShoppingCart.js

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
Let's remove the shopping cart file from this PR.

Reply:
I think it is removed

Reply:
I still see it.  You should be able to delete the file, then do a new commit, and push the branch.

Reply:
I deleted it